po/git-novice.ja.po

```diff
@@ -4294,19 +4276,19 @@ msgid ""
 "on the other hand,\n"
 "moving backward and forward in time becomes much easier."
 msgstr ""
-"The fact that files can be reverted one by one\n"
-"tends to change the way people organize their work.\n"
-"If everything is in one large document,\n"
-"it's hard (but not impossible) to undo changes to the introduction\n"
-"without also undoing changes made later to the conclusion.\n"
-"If the introduction and conclusion are stored in separate files,\n"
-"on the other hand,\n"
-"moving backward and forward in time becomes much easier."
+"ファイルを1つずつ元に戻すことができるという事実は\n"
+"人々が研究を整理する方法を変えるのに役立ちます。\n"
```

Comment:
It's not saying "this helps people change their work style", just that it often has this effect.
```suggestion
"人々が研究を整理する方法を変えることがあります。\n"
```